spec/controllers/spina/registers_controller_spec.rb

```diff
@@ -0,0 +1,157 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Spina::RegistersController, type: :controller do
+  routes { Spina::Engine.routes }
+
+  before do
+    country_data = File.open('./spec/support/country.rsf', &:read)
```

Comment:
The usual way to open a file is `File.read('./spec/support/country.rsf')`.  See https://ruby-doc.org/core-2.4.0/IO.html#method-c-read